### PR TITLE
es: update locale/es/docs/index.md

### PR DESCRIPTION
**Pull request by juliandavidmr**
Grammatical corrections of Spanish language.

### CODE REVIEW

**Comment by nschonni** [comment]
Ping @nodejs/nodejs-es

File: locale/es/docs/index.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Existen diferentes tipos de documentación disponible en este sitio:
 
 ## Referencia de la API
 
-La [Referencia de la API](https://nodejs.org/api/) proporciona información detallada sobre una función ó un objeto en Node.js. Esta documentación indica que argumentos acepta un método, el valor que retorna este método y qué errores pueden estar relacionados al mismo. También indica qué métodos están disponibles para las diferentes versiones de Node.js
+La [Referencia de la API](https://nodejs.org/api/) proporciona información detallada sobre una función ó un objeto en Node.js. Esta documentación indica qué argumentos acepta un método, el valor que retorna este método y qué errores pueden estar relacionados al mismo. También indica qué métodos están disponibles para las diferentes versiones de Node.js.
```
Comment (Tsuna-mi) [inline locale/es/docs/index.md]: Also the 'o' without the accent ;)
```suggestion
La [Referencia de la API](https://nodejs.org/api/) proporciona información detallada sobre una función o un objeto en Node.js. Esta documentación indica qué argumentos acepta un método, el valor que retorna este método y qué errores pueden estar relacionados al mismo. También indica qué métodos están disponibles para las diferentes versiones de Node.js.
```

**Review by Tsuna-mi — APPROVED** [review]
Apart from this little typo about the o character, the rest LGTM.
Thanks @juliandavidmr

**Comment by XhmikosR** [comment]
This seems invalid now, closing.

**Pull request closed without merging**